test/test_legacy_bridge_swap.py

```diff
@@ -0,0 +1,147 @@
+from cosmpy.aerial.contract import LedgerContract
+from gql import gql
+import time, unittest, os, requests, base, decimal, datetime as dt, json
+
+
+class TestContractSwap(base.Base):
+    contract = None
+    amount = decimal.Decimal(10000)
+    denom = "atestfet"
+    db_query = 'SELECT destination, amount, denom from legacy_bridge_swaps'
+
+    def setUp(self):
+        url = "https://github.com/fetchai/fetch-ethereum-bridge-v1/releases/download/v0.2.0/bridge.wasm"
+        if not os.path.exists("../.contract"):
+            os.mkdir("../.contract")
+        try:
+            temp = open("../.contract/bridge.wasm", "rb")
+            temp.close()
+        except:
+            contract_request = requests.get(url)
+            file = open("../.contract/bridge.wasm", "wb").write(contract_request.content)
+
+        self.contract = LedgerContract("../.contract/bridge.wasm", self.ledger_client)
+
+        # TODO: avoid deploying with every test run
+        # Instead, query for active contract and skip if found.
+
+        self.contract.deploy(
+            {"cap": "250000000000000000000000000",
+             "reverse_aggregated_allowance": "3000000000000000000000000",
+             "reverse_aggregated_allowance_approver_cap": "3000000000000000000000000",
+             "lower_swap_limit": "1",
+             "upper_swap_limit": "1000000000000000000000000",
+             "swap_fee": "0",
+             "paused_since_block": 18446744073709551615,
+             "denom": "atestfet",
+             "next_swap_id": 0
+             },
+            self.validator_wallet
+        )
```

Comment:
I think we can drop this `setUp` method. As per our discussions, we've established that we'll be relying on an externally configured legacy bridge via the bridge relayer's docker-compose.yml so that we don't need to deploy any contracts in the test itself.